src/MenuBar.jsx

```diff
@@ -65,16 +65,25 @@ export class homeMenuBar extends React.Component {
       <Navbar collapseOnSelect className={styles.navigationbar}>
         <Navbar.Header>
           <Navbar.Brand className={styles.navbarheader}>
-            <a href="#">Midtown Assistance Center</a>
+            <a href="/dash">Midtown Assistance Center</a>
           </Navbar.Brand>
           <Navbar.Toggle />
         </Navbar.Header>
         <Navbar.Collapse>
           <Nav pullLeft>
-            <NavItem className={styles.navbartext}>About us</NavItem>
+            {window.location.pathname === '/dash/about' ?
+            <MenuItem className={styles.navbartext} href="/dash">Dashboard</MenuItem>
+            :
+            <MenuItem className={styles.navbartext} href="/dash/about">About us</MenuItem>}
```

Comment:
Logic looks correct, just wondering if this is necessary. It might be worth to just show both items at all times. Most websites do that

Reply:
True! Currently I have 'Midtown Assistance Center' also linking to dash but we can discuss adding that today